packages/wrangler/src/pages.tsx

```diff
@@ -10,7 +10,7 @@ import { render } from "ink";
 import Table from "ink-table";
 import { getType } from "mime";
 import React from "react";
-import { format } from "timeago.js";
+import { format as timeagoFormat } from "timeago.js";
```

Comment:
@sidharthachatterjee  I renamed this function because it was otherwise confusing as a generic `format()`